crates/ark/src/lsp/signature_help.rs

```diff
@@ -329,21 +329,12 @@ fn is_within_call_parentheses(x: &Point, node: &Node) -> bool {
         return false;
     };
 
-    let n_children = arguments.child_count();
-    if n_children < 2 {
-        log::error!("`arguments` node only has {n_children} children.");
+    let Some(open) = arguments.child_by_field_name("open") else {
         return false;
-    }
-
-    let open = arguments.child(1 - 1).unwrap();
-    let close = arguments.child(n_children - 1).unwrap();
-
-    if open.node_type() != NodeType::Anonymous(String::from("(")) {
-        return false;
-    }
-    if close.node_type() != NodeType::Anonymous(String::from(")")) {
+    };
+    let Some(close) = arguments.child_by_field_name("close") else {
```

Comment:
I _think_ I'm leaning towards trying to avoid too much usage of `is_missing()` in the semantic paths for now. Hopefully Rowan makes this a non issue, and I don't think it is really going to hurt much as is